e2e/tests/content-history/history.spec.ts

```diff
@@ -0,0 +1,182 @@
+import { test, expect } from '@playwright/test';
+import { login } from '../../utils/login';
+import { resetDatabaseAndImportDataFromPath } from '../../scripts/dts-import';
+import { describeOnCondition } from '../../utils/shared';
+
+const hasFutureFlag = process.env.STRAPI_FEATURES_FUTURE_CONTENT_HISTORY === 'true';
+
+describeOnCondition(hasFutureFlag)('History', () => {
+  test.describe('Collection Type', () => {
```

Comment:
I would add one more important aspect: checking that the history is isolated between the different locales.

At the end of the test, switch to a different locale, create something, save, go to history, and check that we only see this change on the new locale and not the stuff we did before on english content

Reply:
Good idea! 👍

Reply:
Currently Single types throw an error when creating and entry in another locale so I'll only add the test for collection types right now and add a TODO for single types